resources/views/Logistics/index.blade.php

```diff
@@ -1,7 +1,50 @@
 <x-app-layout>
     <x-slot name="header">
         <h2 class="font-semibold text-xl text-gray-800 leading-tight">
-            {{ __('Logistics') }}
+            {{ __('Logistics Dashboard') }}
         </h2>
     </x-slot>
+
+    <div class="bg-stockhive-grey-dark text-white shadow-sm rounded-lg mt-8 lg:w-[85%] w-full m-auto p-4">
+        <div class="flex justify-between items-center gap-8 my-4 border-grey bg-stockhive-grey rounded-lg p-4 border-2 m-auto w-[90%] text-right">
+            <h1 class="text-lg font-semibold">Logistics Dashboard</h1>
+            </a>
+        </div>
```

Comment:
Remove this title it's duplicated and doesn't follow the rest of the site theme.